tools/pylib/boutdata/restart.py

```diff
@@ -341,7 +329,7 @@ def is_pow2(x):
 
 
 
-def expand(newz, path="data", output="./", informat="nc", outformat=None):
+def resizeZ(newNz, path="data", output="./", informat="nc", outformat=None):
```

Comment:
I suspect this is not used in an automatic script anywhere, and the name is much better